controller/internal/enforcer/applicationproxy/http/http.go

```diff
@@ -182,6 +186,30 @@ func (p *Config) GetClientCertificateFunc(_ *tls.CertificateRequestInfo) (*tls.C
 	p.RLock()
 	defer p.RUnlock()
 	if p.cert != nil {
+		cert, err := x509.ParseCertificate(p.cert.Certificate[0])
+		if err != nil {
+			zap.L().Error("http: Cannot build the cert chain")
```

Comment:
I'd like to make this error different from the next one @ L197 -- if it shows up in a log we can quickly see where it came from. 
"http: Cannot parse the certificate"

Reply:
that would indeed be better